### ISSUE TITLE
Blurred pictures

### ISSUE THREAD

**Issue by PavanLuca** [TGX-Android/Telegram-X#483]
Picture on my file manager.
![Screenshot_2023-10-18-23-02-25](https://github.com/TGX-Android/Telegram-X/assets/123581658/8532aadd-407b-4b19-942d-c6a7b9e6b229)

Picture as attachment on a message.
![Screenshot_2023-10-18-23-01-14](https://github.com/TGX-Android/Telegram-X/assets/123581658/d2955237-133d-4076-9fa1-c862948c849d)

**Comment by matinlunaire** [comment]
Yep, having this too on Android 6. I initially suggested this would be a scaling issue due to the uncommonly low resolution of my device's screen, but seems not to be the case.

**Comment by PavanLuca** [comment]
Looks like that Telegram X has some difficulties on locating the picture and loads its thumbnail instead of the original picture at its full resolution 😁.

**Comment by PavanLuca** [comment]
Looks like that Telegram X has some difficulties locating the picture and loads its thumbnail instead of the original picture at its full resolution 😁.
